### ISSUE TITLE
 Payment Processor Error message :8000: Couldn't create SEPA mandate. Error was: creditor_id is not a valid integer 

### ISSUE THREAD

**Issue by StijnVanhandsaeme** [Project60/org.project60.sepa#523]
I found similar errors in  the issues her but not this one and I don't see a possible fix that would help me. So: when I install civisepa (v. 1.2.5 on civicrm 5.8.2 but I also tried 1.2.6.beta3) I get this error:  Payment Processor Error message :8000: Couldn't create SEPA mandate. Error was: creditor_id is not a valid integer 
Locally I re-installed completely and used the test data from the install but still the same problem... I might be missing something very basic (not to use the word stupid :-))... but I can't seem to find a place to start looking as I don't see how the creditor_id should be an integer... if that is the SEPA ID.

**Comment by ErikHommel** [comment]
Did you set a default creditor in the SEPA settings?

**Comment by StijnVanhandsaeme** [comment]
Had to check but yes, we did. Thank you for thinking with me

**Comment by StijnVanhandsaeme** [comment]
Maybe this might help in debugging: switched on error logging and I see this in de drupal watchdog:
`Notice: Undefined property: CRM_Core_Payment_SDD::$_paymentForm in CRM_Core_Payment->getForm() (line 487 of /sites/all/modules/civicrm/CRM/Core/Payment.php).`

**Comment by StijnVanhandsaeme** [comment]
OK. We got it  'working'. I took over a configuration where on /civicrm/admin/paymentProcessor in the field "SEPA Creditor identifier" had the SEPA creditor identifier (BE....) filled in (sounds logical, no?). Replacing that with the contact id solved the issue. 
So my question is actually: what should be filled in the SEPA Creditor identifier field on /civicrm/admin/paymentProcessor, what does it do?
